src/agent/state.rs

```diff
@@ -1,6 +1,7 @@
 pub const STORAGE_KEY_CSRF_TOKEN: &str = "ctron/oauth2/csrfToken";
 pub const STORAGE_KEY_LOGIN_STATE: &str = "ctron/oauth2/loginState";
 pub const STORAGE_KEY_REDIRECT_URL: &str = "ctron/oauth2/redirectUrl";
+pub const STORAGE_KEY_CURRENT_URL: &str = "ctron/oauth2/currentUrl";
```

Comment:
I'd consider this key name misleading, because at the moment when user actually reads it, it's already not the "Current url"

When I was playing with my own implementation, I considered something like `STORAGE_KEY_POST_LOGIN_REDIRECT_URL` or `STORAGE_KEY_ORIGINAL_URL`

Reply:
Hi @kate-shine , I can rename it. It is not exposed (should not once we have a way to retrieve it via the client api or the handling of the local redirection by the component itself) to the outside world and I named it after the method I get it from `self::get_current_url()`.

Reply:
I guess we should probably name the all local redirection as a process and have a way to 'activate' it as an extra local action when being redirected from the Idp.

Reply:
I would also prefer a name like "post login redirect", as that makes is pretty clear what the intention is. Also, feel free (if you like) to change the prefix from `ctron` … seeing that now, I think I should have done that some time ago :rofl:

Reply:
Depends on how we want to see it? Do we want to name it as how we want to use it, or do we want to name it as what it is. It is the url of the ressource that initiated the flow but indeed we want to automatically redirect to it once the login is complete. Today the component does not do it automatically. So from a developer point of view it is the url the user tried to access before the authentication start and it is up to the dev to perform the Post login redirection.

Reply:
I will rename it to post login url but for me this sound strange... Cause it gives the impression this post url is settable which is not. It is the original url. It is what we do with it that makes it a PostRedirectUrl. If you both agree on that. Let's do it :)

Reply:
Indeed if we manage to have everything handle internally, postLoginUrl makes full sens. My bad. :)